README.md

```diff
@@ -78,13 +78,15 @@ This will set up a new application on your Heroku account using this repo as a t
 
 -->
 
-### Import to Replit
+### Import to CodeSandbox
 
-Create a Replit account at [Replit - Sign up](https://replit.com/signup), and then click on this button:
+1. Log in or register on [CodeSandbox - Sign in](https://codesandbox.io/signin)
 
-[![Run on Replit](https://replit.com/badge?caption=Import%20to%20Replit)](https://replit.com/github/upleveled/express-guest-list-api-memory-data-store)
+2. Change the `express-guest-list-api-memory-data-store` repo URL to include "box" (`githubbox.com` instead of `github.com`) and hit return. This will open the project on CodeSandbox
```

Comment:
```suggestion
2. Change the `express-guest-list-api-memory-data-store` repo URL to include "box" (`githubbox.com` instead of `github.com`) and hit return - this will open the project on CodeSandbox
```